### PR TITLE
Add cache client function deployment support

### PR DESCRIPTION
**Pull request by octonawish-akcodes**
Adding cache client function deployment support in cloud script. Need reviews @spbnick

### CODE REVIEW

File: cloud [hunk 1]
```diff
@@ -1918,9 +1938,10 @@ function execute_command() {
             --load-queue-trigger-topic="$load_queue_trigger_topic" \
             --pick-notifications-trigger-topic \
               "$pick_notifications_trigger_topic" \
+            --updated-urls-topic="$updated_urls_topic" \
             --updated-topic="$updated_topic" \
             --spool-collection-path="$spool_collection_path" \
-            --env-yaml="${env_yaml}"
+            --env-yaml="${env_yaml}"
```
Comment (tales-aparecida) [inline cloud:1944]:
```suggestion
            --env-yaml="${env_yaml}"
```

Remove trailing space

File: cloud [hunk 2]
```diff
@@ -1170,6 +1170,7 @@ function pubsub_subscription_withdraw() {
 #       --updated-topic=NAME
 #       --updated-debug-subscription=NAME
 #       --pick-notifications-trigger-topic=NAME
+#       --cache-topic=NAME
```
Comment (tales-aparecida) [inline cloud:1173]: 🤔 why is this different?

Comment (octonawish-akcodes) [reply cloud:1173]: Ahh forgot to update the comments, thank you for pointing let me update it :)

File: cloud (hunk 1 above)
Comment (octonawish-akcodes) [reply cloud:1944]: Ah okay

**Comment by tales-aparecida** [comment]
Have you executed any manual tests with these changes? Can you write a little bit about them?

**Comment by octonawish-akcodes** [comment]
> Have you executed any manual tests with these changes? Can you write a little bit about them?

The changes this PR is doing are as follows:
- First, it will add support for deploying the kcidb_cache_urls using the cloud script.
- It also adds support for that cloud function's PUB/SUB topics so that the data is stored in the cloud storage bucket whenever a new message is published.

Below is the visualization of the tests I performed:

The project is being deployed.
Screenshot from 2023-07-13 23-38-25

Then a message gets published.
Screenshot from 2023-07-13 23-40-15

then the cloud function tries to fetch the file .

**Comment by spbnick** [comment]
Thank you, @octonawish-akcodes, and @tales-aparecida!

**Pull request merged**